integrationExamples/gpt/blueconicRtdProvider_example.html

```diff
@@ -0,0 +1,122 @@
+<html>
+<head>
+    <script>
+      var testBlueconicData = `{"bc_segments": ["allvisitors"], "bidId": "1234", "identifier": ['test','123']}`;
```

Comment:
The single quotes around test and 123 are making this fail parseJson in your test page. Just swapping them out with double quotes seemed to work.

Reply:
Done